extensions/replication/tasks/ReplicateObject.js

```diff
@@ -305,6 +303,22 @@ class ReplicateObject extends BackbeatTask {
                 return cb(errors.BadRole);
             }
             return cb(null, roles[0], roles[1]);
+        })
+        .catch(err => {
```

Comment:
nit: using utils.callbackify() instead of promise.then() allows to keep the same exact structure as before... or you can at least put the `.catch` block before `.then`